### ISSUE TITLE
[feature request] tab needed in order to open several pdf in one window

### ISSUE THREAD

**Issue by KDB2** [linuxmint/xreader#4]
Dear team,

it would be really helpful to use tabs in xreader. It would allow to open several pdf in one instance of xreader. 

Thanks,
KDB

**Comment by anandrkris** [comment]
Yes, helpful feature. Remember it from my college days when I used Foxit reader.

**Comment by Cos-mo** [comment]
I agree, but there should be an option to open in a new window in case you want to see 2 documents or 2 opened copies of a document die by side. Similar as you can do this in gedit / xed.

**Comment by korneljahn** [comment]
@JosephMcc Please add the FEATURE REQUEST label (and the tag [feature request] can then be deleted from the title).

**Comment by korneljahn** [comment]
@JosephMcc Thanks

**Comment by Laurencio** [comment]
I wish too we had tab support on xreader. And as Cos-mo said, the availability to open in other tabs or windows (just as web browsers do)
I find this is really important when not reading a book but working with a lot of small documents in which the secuence of them is important. (Common Alt+Tab switching is useless in this case)
So i had to switch to qpfview or okular. But this readers work very different with the selection tool (they draw a frame into an area) and i need a selection behaviour like Evince and Xreader have.
In other words, Evince with tab support would be perfect for my work needs, but the Gnome developers are known to reject the tab idea. Hope xreader devs are more open to this.

**Comment by mickaelalbertus** [comment]
I try to implement this functionality but we need to think to the new behaviour. For example: how to store the size of the window for the document to be restored? One idea is to not save anymore this metadata? 
Does the application open in a new tab by default? Maybe I will have other questions like that!

**Comment by tschertel** [comment]
Any news on this request?? I'd like to see that implemented too.
Thanks for all your work on this amazing app :)

**Comment by svastian99** [comment]
Please support display in tabs  {something like qpdfview}

**Comment by haggen88** [comment]
it would be very helpful to have this functionality in xreader

**Comment by Laurencio** [comment]
> I try to implement this functionality but we need to think to the new behaviour. For example: how to store the size of the window for the document to be restored? One idea is to not save anymore this metadata?
> Does the application open in a new tab by default? Maybe I will have other questions like that!
I'm not sure if I get you well.
But you may check out qpdfview or okular and see how they deal with that things.
Qpdfview is great because it is very customizable, though it lacks some funtionality, e.g. the selecting text feature is not certainly standard.
But think that if you get this working you'll make like the first  tabbed reader in the gtk world. Otherwise evince, atril and xreader are so the same...

**Comment by areographe** [comment]
Just checking in, is there any chance we could see `xed` style tabbed browsing in Xreader? I much prefer it to other reader software, but as I'm studying city planning I have to look at large numbers of (10+) small PDF files for different sections and clauses, and as @Laurencio mentioned the sequence is important, so <kbd>Alt</kbd><kbd>Tab</kbd> won't work. I am using Okular due to its tab support, but would like to switch back to Xreader, as it works great in every other respect.

**Comment by jkwnki** [comment]
Still missing :(

**Comment by Alonso666cr** [comment]
Hello! I'd like to express my strong interest in this feature request.

As a professional, I frequently need to 
reference multiple PDF documents simultaneously. Currently, I have to 
choose between:
- Using multiple Xreader windows (inefficient)
- Installing heavier alternatives like Okular (259 MB of dependencies)
- Using qpdfview (70 MB of Qt6 dependencies)

Why Xreader with tabs would be valuable:
- Maintains the lightweight nature of Xreader (~10-20 MB)
- Uses GTK (already in the system)
- Would benefit the entire Linux Mint community

I'd be happy to help test this feature if development begins.

If you consider it I would like to help you staring a campaign in Patreon to catch resources to pay that work. 

Thank you for considering this 9-year-old request! 👍
